middleware/packet-forward-middleware/router/ibc_middleware.go

```diff
@@ -135,6 +136,17 @@ func getBoolFromAny(value any) bool {
 	return boolVal
 }
 
+// getReceiver returns the receiver address for a given channel and original sender.
+// it overrides the receiver address to be a hash of the channel/origSender so that
+// the receiver address is deterministic and can be used to identify the sender on the
+// initial chain.
+func getReceiver(channel string, originalSender string) (string, error) {
+	senderStr := fmt.Sprintf("%s/%s", channel, originalSender)
```

Comment:
Should we also use the port here? (in case this is used to wrap a port other than `transfer` in the future)

Reply:
^ this makes sense to me

Reply:
This is not necessary, since PFM is ICS-20 middleware. Also, channels are on top of a port in IBC, so in the case this was modified in the future to support non-transfer packets, the channel ID would still give us enough to cover the channel+port. https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel